internal/servers/controller/handlers/sessions/session_service.go

```diff
@@ -196,6 +216,8 @@ func (s Service) authResult(ctx context.Context, id string, a action.Type) auth.
 			res.Error = handlers.NotFoundError()
 			return res
 		}
+	case action.ListSelf:
```

Comment:
I think this is just `case action.List, action.ListSelf` above?

Reply:
Yes, I don't know what happened to this file. I thought I had fixed it up beyond this prior to sending out the PR.  Sorry.  I've fixed this and the other places in this file where things are obviously wrong.  PTAL.